sdk/dotnet/GkeHub/FeatureMembership.cs

```diff
@@ -33,25 +33,25 @@ public partial class FeatureMembership : Pulumi.CustomResource
         /// Config Management-specific spec.
         /// </summary>
         [Output("configmanagement")]
-        public Output<Outputs.FeatureMembershipConfigmanagement?> Configmanagement { get; private set; } = null!;
+        public Output<Outputs.FeatureMembershipConfigmanagement> Configmanagement { get; private set; } = null!;
```

Comment:
Need to check whether this is a (technically) breaking change.  May be worth noting in the changelog.

Reply:
Same comments apply to other outputs in this file that were change in this PR.

Reply:
Per Zoom convo with @stack72, these are upstream in both the TF provider and the GCP API.